config/test.exs

```diff
@@ -41,6 +41,13 @@ config :logger, level: :warn
 # Initialize plugs at runtime for faster test compilation
 config :phoenix, :plug_init_mode, :runtime
 
+# Agent heartbeat interval. Adding one extra second to the agent 5s interval to avoid glitches
+config :trento, Trento.Heartbeats, interval: :timer.seconds(6)
```

Comment:
This has the same exact value that in `config.exs`. What's the purpose?

Reply:
To decouple the test environment from the production environment. If this didn't exist in `config/test.exs` and someone were to update the value in `config/config.exs`, then the test might start failing.

Reply:
Okey, defensive measure I guess